Kiidle/project.feeder.backend
# Edit Feed

## Description
This change adds the ability to edit a feed. With the FeedUpdateView, users can modify the text of an existing feed.

## Changes
- Add the FeedUpdateView class in feeds/views.py, which provides a form page for editing a feed.
- Add URLs for the FeedUpdateView in feeds/urls.py.

## Testing Instructions
- Clone this Pull Request and run it on a testing environment.
- Open the page /feeds/<id>/update in your browser.
- Verify that the form for editing the feed is displayed.
- Modify the text of the feed and submit the form.
- Verify that the feed was updated and that you were redirected to the feed detail page.
- Verify that the text of the feed was updated on the feed detail page.

## Screenshot
<img width="393" alt="Bildschirm­foto 2023-03-28 um 09 38 59" src="https://user-images.githubusercontent.com/104766564/228163632-02bbaf89-4ac3-412f-8a28-9f6207c92777.png">
